plugins/modules/ovirt_qos.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ovirt_qos
+short_description: "Module to manage QoS entries in oVirt/RHV"
+author:
+- "Niall O Donnell (niall.odonnell@ppb.com)"
+description:
+    - "Module to manage QoS entries in oVirt/RHV."
+    - "Doesn't support updating a QoS that exists"
+    - "Only works with storage QoS entries atm"
+options:
+    id:
+        description:
+            - "ID of the QoS to manage. Either C(id) or C(name) is required."
+        type: str
+    name:
+        description:
+            - "Name of QoS to manage. Either C(id) or C(name)/C(alias) is required."
+        type: str
+    description:
+        description:
+            - "Description of the QoS."
+        type: str
+    data_center:
+        description:
+            - "Name of the data center where the QoS entry should be created."
+        type: str
+        max_iops=dict(default=None),
+        read_iops=dict(default=None),
+        write_iops=dict(default=None),
+        max_throughput=dict(default=None),
+        read_throughput=dict(default=None),
+        write_throughput=dict(default=None),
```

Comment:
Please remove these lines the documentation is not compiled properly because of it.